src/MeetingsSDKAdapter.js

```diff
@@ -895,6 +904,70 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     return state$;
   }
 
+  /**
+   * Attempts to toggle settings to the given meeting ID.
```

Comment:
Actually, this operation can't fail, so it's not an attempt like "join meeting", "mute audio", which depend on external factors and may fail.

Maybe: 
> Toggles the showSettings flag of the given meeting.